trustpinning/trustpin.go

```diff
@@ -60,6 +63,7 @@ func NewTrustPinChecker(trustPinConfig TrustPinConfig, gun string) (CertChecker,
 	}
 
 	if !trustPinConfig.DisableTOFU {
+		logrus.Debugf("Trust-pinning: allowing TOFU")
```

Comment:
Nitpick:  Would it make sense to phrase this similarly to the trust pinning using CA bundle log?  `"Trust-pinning using TOFU"` or some such?